src/pages/CategoriesPage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Header from '../components/Header';
+
+const Categories = () => (
+  <div>
+    <Header />
```

Comment:
- [ ] [OPTIONAL] In the spirit of keeping your code DRY, the `Header` component can be included in the `App.js` since it is shared across pages.